Newsletter2Go/Export/Setup/UpgradeData.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Newsletter2Go\Export\Setup;
+
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Integration\Model\ConfigBasedIntegrationManager;
+use Magento\Framework\Setup\UpgradeDataInterface;
+
+class UpgradeData implements UpgradeDataInterface
+{
+    /**
+     * @var ConfigBasedIntegrationManager
+     */
+    private $integrationManager;
+
+    /**
+     * @param ConfigBasedIntegrationManager $integrationManager
+     */
+    public function __construct(ConfigBasedIntegrationManager $integrationManager)
+    {
+        $this->integrationManager = $integrationManager;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $this->integrationManager->processIntegrationConfig(['Newsletter2Go Integration']);
```

Comment:
The string can be replaced with `RegisterIntegration::NEWSLETTER2GO_INTEGRATION_NAME`.